src/utils/postProcessResults.ts

```diff
@@ -83,6 +82,14 @@ const postProcessResults = (
               return true;
             }
             return columnFilter.value.some((v) => r[columnFilter.key] === v);
+          case "greater than":
+          case "less than":
+            const transformedFilter = transform(columnFilter.value[0]);
+            const transformedResult = transform(r[columnFilter.key]);
+
+            return columnFilter.type === "greater than"
+              ? transformedResult > transformedFilter
+              : transformedResult < transformedFilter;
```

Comment:
we are already checking the `columnFilter.type === "greater than"` case, so I typically keep each "function" in each case